Gemfile.lock.release

```diff
@@ -1496,4 +1510,4 @@ DEPENDENCIES
   yard (>= 0.9.36)
 
 BUNDLED WITH
-   2.5.5
+   2.5.10
```

Comment:
is this why we're getting darwin gems?

Reply:
partly, yes - technically this happened in 2.5.5 also, but there were bugs with it (which is why we need to go to 2.5.10 anyway)